GSoC2019/README.md

```diff
@@ -26,10 +23,14 @@ Everything is inside the src folder.
  
 ## Local deploy
 
-Just access the index.html file in Firefox browser (in Chrome there is a restriction that requires accessing files only through HTTP or HTTPS).
+The files can be hosted locally by running the following command in the `GSoC2019/src` folder.
+```
+python -m http.server 4000

```

Comment:
Do users need to install a module for this? I tried on my computer but nothing happened. I use python 3

Reply:
I don't think any new modules need to be installed as the `http` module is present in the [Python Standard Library](https://docs.python.org/3/library/) under [Internet Protocols and Support](https://docs.python.org/3/library/internet.html). The above command is also working for the directory on a fresh installation of Fedora that I installed a few days back. 
I wonder what is not working in you case, though :confused:

Reply:
Ok sorry, now it works.